src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -258,7 +254,7 @@ private Path getGameDataDirectory(OperatingSystem os, Path settingsGameDataDirec
                     BundleUtils.getLabel("message_dialog_title_chooseGameDataDirectory"));
             if (Files.notExists(gameDataDirectory)) {
                 logger.info("The new game data directory is not approved. The TerasologyLauncher is terminated.");
-                Platform.exit();
+                javafx.application.Platform.exit();
```

Comment:
That's not code I actually touched... It's only listed here because due to the renaming of `OperatingSystem` to `Platform`, the `Platform` in `Platform.exit()` became ambiguous.